.github/workflows/development.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       REACT_APP_ASSETS_PATH: ${{vars.ASSETS_PATH}}
       REACT_APP_ANDROID_STORE_PATH: ${{vars.ANDROID_STORE_PATH}}
       REACT_APP_MATRIX_HOME_SERVER_URL: ${{vars.MATRIX_HOME_SERVER_URL}}
-      REACT_APP_ZNS_EXPLORER_URL: ${{vars.ZNS_EXPLORER_URL}}
+      REACT_APP_ZNS_EXPLORER_URL: ${{vars.REACT_APP_ZNS_EXPLORER_URL}}
```

Comment:
Would have been better to fix the github var to be consistent with the pattern of the other variables.

Reply:
Yeah agreed, I also need to remove leading '/' on the sepolia explorer path so I'll update these this afternoon as well